### DISCUSSION TITLE
How APP receives and sends packets

### DISCUSSION THREAD

**Discussion by satishsept7** [CloudNativeDataPlane/cndp#149] [Q&A] [answered]
Hi,
I want to use cndp for POC activity. Need to know what is interface with which applications can receive packet from CNDP and how those interfaces between application and CNDP can be configured. What is API to send and receive packets from CNDP for fastpath.

**Comment by jeffreybshaw** [comment, marked as answer]
Hello, you probably want to start with the [documentation](https://cndp.io/guide/index.html). An example to receive and transmit packets is shown in [cndpfwd](https://github.com/CloudNativeDataPlane/cndp/blob/main/examples/cndpfwd/main.c#L150).
